src/containers/Gurb/requirementsValidations.js

```diff
@@ -0,0 +1,25 @@
+import * as Yup from 'yup'
+
+export const lightValidations = Yup.object().shape({
+  has_light: Yup.string()
```

Comment:
It make sense to validate light-on/light-off and selfconsumption-on/selfconsumption-off, when the expected requirement values are light-on and selfconsumption-off?

Reply:
Here we are validating that the user has to choose one of the allowed options (if has light or if has no light). If the user doesn't choose one of the options, it can not go any further and the next button stays un-clickable.